src/modules/goal/containers/GoalPage.js

```diff
@@ -11,14 +11,14 @@ import { fetchCurrentGoalState } from "../actions";
 
 export class GoalPage extends Component {
   componentWillMount() {
-    this.props.fetchData(this.props.user.apiToken);
+    this.props.fetchData(this.props.user.apiToken, this.props.team.id);
   }
 
   render({ currentAmount, nextAmount, nextText }) {
     return (
       <Page id="goalPage">
         <Header>
-          <h1>{I18n.t("goal.title")}</h1>
+          <h1>{this.props.team.name} {I18n.t("goal.title")}</h1>
```

Comment:
You can use deconstruction in the render function `render({ team })` and write `team.name` here instead of `this.props.team.name`